src/workload.ts

```diff
@@ -23,6 +23,13 @@ export interface WorkloadProps extends pod.AbstractPodProps {
    */
   readonly select?: boolean;
 
+  /**
+   * Automatically spread pods across hostname and zones.
+   *
+   * @see https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#internal-default-constraints
+   * @default true
```

Comment:
The default should be `false` to be more developer friendly, i.e allowing default constructs to run on a dev cluster with one node.